app.py

```diff
@@ -23,14 +30,70 @@ def index():
     return render_template('pages/index.html')
 
 
-@app.route("/user_registration")
+@app.route("/user_registration", methods=["GET", "POST"])
 def user_registration():
+    if request.method == "POST":
+
+        existing_user = mongo.db.users.find_one(
+            {"full_name": request.form.get("full_name").lower()})
```

Comment:
Hi Gemma,

Firstly, this work is fantastic! 🤜 Very impressive. 
My only feedback is should we be checking for email, rather than full_name here? An email would be more unique in the grand scheme of things then a name. As in Waterford here, there are literally 100 people with the same name and even nickname 😂😂😂

What do you think? 😄